Modules/gcmodule.c

```diff
@@ -962,6 +962,24 @@ clear_freelists(void)
     (void)PyContext_ClearFreeList();
 }
 
+// Show stats for objects in each gennerations.
+static void
+show_stats_each_generations(struct _gc_runtime_state *state)
+{
+    char buf[100];
+    size_t pos = 0;
+
+    for (int i = 0; i < NUM_GENERATIONS && pos < sizeof(buf); i++) {
+        pos += PyOS_snprintf(buf+pos, sizeof(buf)-pos,
+                             " %zd", gc_list_size(GEN_HEAD(state, i)));
+    }
+
+    PySys_WriteStderr(
```

Comment:
We need to use `PySys_FormatStderr`  if `%zd` format is not portable yet.
I sent mail about it to Python-dev.